tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_entry_exit_criteria.py

```diff
@@ -0,0 +1,349 @@
+        #   If the difference b/w updated count of osds and old osd count is not 3 then expansion failed
+        assert number_of_osds_added == 3, "Exit criteria verification FAILED: osd count mismatch"
+
+        logging.info("Exit criteria verification Success: Correct number of OSDs are added")
+
+        # The newly added capacity takes into effect at the storage level
+        ct_pod = pod_helpers.get_ceph_tools_pod()
+        output = ct_pod.exec_ceph_cmd(ceph_cmd='ceph osd df')
+        total_space_after_expansion = int(output.get('summary').get('total_kb'))
+        osd_size = int(output.get('nodes')[0].get('kb'))
+        expanded_space = osd_size * 3  # 3 OSDS are added of size = 'osd_size'
+        logging.info(f"space output == {output} ")
+        logging.info(f"osd size == {osd_size} ")
+        logging.info(f"total_space_after_expansion == {total_space_after_expansion} ")
+        expected_total_space_after_expansion = total_space_b4_expansion + expanded_space
+        logging.info(f"expected_total_space_after_expansion == {expected_total_space_after_expansion} ")
+        assert total_space_after_expansion == expected_total_space_after_expansion, \
+            "Exit criteria verification FAILED: Expected capacity mismatch"
+
+        logging.info("Exit criteria verification Success: Newly added capacity took into effect")
+
+        # IOs should not stop, No OCP/OCS nodes should go to NotReady state,No OCP/OCS nodes should
+        # go to NotReady state
+        if status_noobaa_s3_io.result(timeout=900) and node_status.result(timeout=900):
+            for cluster_ios in status_cluster_ios:
+                assert cluster_ios.result(timeout=900), \
+                    "Exit criteria verification FAILED: IOs did not complete"
+
+
+        logging.info("Exit criteria verification Success: IOs completed successfully")
+
+        # 'ceph osd tree' should show the new osds under right nodes/hosts
+        #   Verification is different for 3 AZ and 1 AZ configs
+        ct_pod = pod_helpers.get_ceph_tools_pod()
+        tree_output = ct_pod.exec_ceph_cmd(ceph_cmd='ceph osd tree')
+        logging.info(f"### OSD tree output = {tree_output}")
+        if config.ENV_DATA['platform'] == 'vsphere':
+            assert cluster_helpers.check_osd_tree_1az_vmware(tree_output, len(osd_pods_after)),\
+                "Exit criteria verification FAILED: Incorrect ceph osd tree formation found"
+
+        aws_number_of_zones = 3
+        if config.ENV_DATA['platform'] == 'AWS':
+            # parse the osd tree. if it contains a node 'rack' then it's a AWS_1AZ cluster. Else, 3 AWS_3AZ cluster
+            for i in range(len(tree_output['nodes'])):
+                if tree_output['nodes'][i]['name'] in "rack":
+                    aws_number_of_zones = 1
+            if aws_number_of_zones == 1:
+                assert cluster_helpers.check_osd_tree_1az_aws(output, len(osd_pods_after)), \
```

Comment:
I think that need to replace here and in line 315 the variable 'output' with 'tree_output'